cloudmarker/events/azsqldatabasedisabledtdeevent.py

```diff
@@ -0,0 +1,95 @@
+"""Microsoft Azure SQL DB Transparent Data Encryption (TDE) event.
+
+This module defines the :class:`AzSQLDatabaseDisabledTDEEvent` class that
+identifies if a SQL Database has TDE disabled . This plugin works on the
+SQL DB properties found in the ``ext`` bucket of ``sql_db`` records.
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class AzSQLDatabaseDisabledTDEEvent:
+    """Az SQL Database TDE event plugin."""
```

Comment:
Let us start spelling out `Az` in full as `Azure` because this is going to be part of publicly visible documentation.

If you would like to make the same change in other plugins, that would be acceptable as part of this pull request but it is not necessary.

I think the `Database` here should also be in lowercase like `database` because we are using it just like any other common word (not really a brand name).

Reply:
Addressed